dist/.goreleaser.yaml

```diff
@@ -196,7 +208,6 @@ dockers:
       - client
     image_templates:
       - "scylladb/scylla-manager:{{ .Version }}"
-      - "scylladb/scylla-manager:latest"
```

Comment:
Why is this removed?

Reply:
Latest tag is related only to latest version which we deal during promotion . it's not needed here

Reply:
Sure, makes sense. Please update the commit description to explain this.

Reply:
Updated